### ISSUE TITLE
Chart does not show anything when all the data are equal

### ISSUE THREAD

**Issue by USBA** [pichukov/LightChart#2]
The chart does not show anything if all the elements in the array is same

Ex: 
```
LightChartView(data: [10, 10, 10, 10, 10],
                               type: .curved,
                               visualType: .filled(color: chartColor, lineWidth: 5),
                               currentValueLineType: .dash(color: .gray, lineWidth: 1, dash: [5]))
``` 

I've even tried adding an **offset**, but the result is still same
```
LightChartView(data: [10, 10, 10, 10, 10],
                               type: .curved,
                               visualType: .filled(color: chartColor, lineWidth: 5),
                               offset: 0.2,
                               currentValueLineType: .dash(color: .gray, lineWidth: 1, dash: [5]))
```

**Current behavior**

- The chart does not show anything if all the data are same

**Expected behavior**

- The chart shows a horizontal line if all the data are same

Please fix this. 
Thanks so much for this amazing SwiftUI library.

**Comment by pichukov** [comment]
Thanks for raising this issue. Now it should be fixed from version `1.0.1`

**Comment by USBA** [comment]
Thanks for quickly fixing the issue 😊
